[freetype] Build error

Package: freetype[brotli,bzip2,core,error-strings,png,zlib]:x64-windows -> 2.12.1#2

**Host Environment**

- Host: x64-windows
- Compiler: MSVC 19.34.31935.0
-    vcpkg-tool version: 2022-11-10-5fdee72bc1fceca198fb1ab7589837206a8b81ba
    vcpkg-scripts version: 163fe7bd3 2022-12-02 (2 days ago)

**To Reproduce**

`vcpkg install `

**Failure logs**

```
-- Using cached freetype-2-6482de17.12.1.tar.xz.
-- Extracting source C:/usr/vcpkg/downloads/freetype-2-6482de17.12.1.tar.xz
-- Applying patch 0003-Fix-UWP.patch
-- Applying patch brotli-static.patch
-- Applying patch bzip2.patch
-- Applying patch fix-exports.patch
-- Applying patch error-strings.patch
-- Using source at C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean
-- Found external ninja('1.11.0').
-- Configuring x64-windows
CMake Error at scripts/cmake/vcpkg_execute_required_process.cmake:96 (message):
    Command failed: "C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" -v
    Working Directory: C:/usr/vcpkg/buildtrees/freetype/x64-windows-rel/vcpkg-parallel-configure
    Error code: 1
    See logs for more information:
      C:\usr\vcpkg\buildtrees\freetype\config-x64-windows-CMakeCache.txt.log
      C:\usr\vcpkg\buildtrees\freetype\config-x64-windows-CMakeCache.txt.log
      C:\usr\vcpkg\buildtrees\freetype\config-x64-windows-out.log

Call Stack (most recent call first):
  C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/share/vcpkg-cmake/vcpkg_cmake_configure.cmake:247 (vcpkg_execute_required_process)
  ports/freetype/portfile.cmake:31 (vcpkg_cmake_configure)
  scripts/ports.cmake:147 (include)



```
<details><summary>C:\usr\vcpkg\buildtrees\freetype\config-x64-windows-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: c:/usr/vcpkg/buildtrees/freetype/x64-windows-rel
# It was generated by CMake: C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//Path to a library.
BROTLICOMMON_LIBRARIES:FILEPATH=BROTLICOMMON_LIBRARIES-NOTFOUND

//Path to a file.
BROTLIDEC_INCLUDE_DIRS:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include

//Path to a library.
BROTLIDEC_LIBRARIES:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/brotlidec.lib

//No help, variable specified on the command line.
BUILD_SHARED_LIBS:UNINITIALIZED=ON

//Path to a file.
BZIP2_INCLUDE_DIR:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include

//Path to a library.
BZIP2_LIBRARY_DEBUG:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/bz2d.lib

//Path to a library.
BZIP2_LIBRARY_RELEASE:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/bz2.lib

//Path to a program.
CMAKE_AR:FILEPATH=C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/lib.exe

//Choose the type of build, options are: None Debug Release RelWithDebInfo
// MinSizeRel ...
CMAKE_BUILD_TYPE:STRING=Release

CMAKE_CROSSCOMPILING:STRING=OFF

CMAKE_CXX_FLAGS:STRING=' /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP '

CMAKE_CXX_FLAGS_DEBUG:STRING='/D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 '

CMAKE_CXX_FLAGS_RELEASE:STRING='/MD /O2 /Oi /Gy /DNDEBUG /Z7 '

//C compiler
CMAKE_C_COMPILER:FILEPATH=C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/cl.exe

CMAKE_C_FLAGS:STRING=' /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP '

CMAKE_C_FLAGS_DEBUG:STRING='/D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 '

//Flags used by the C compiler during MINSIZEREL builds.
CMAKE_C_FLAGS_MINSIZEREL:STRING=/MD /O1 /Ob1 /DNDEBUG

CMAKE_C_FLAGS_RELEASE:STRING='/MD /O2 /Oi /Gy /DNDEBUG /Z7 '

//Flags used by the C compiler during RELWITHDEBINFO builds.
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=/MD /Zi /O2 /Ob1 /DNDEBUG

//Libraries linked by default with all C applications.
CMAKE_C_STANDARD_LIBRARIES:STRING=kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib

//No help, variable specified on the command line.
...
Skipped 261 lines
...

// found after toolchain/system libraries/packages.
VCPKG_PREFER_SYSTEM_LIBS:BOOL=OFF

//Enable the setup of CMAKE_PROGRAM_PATH to vcpkg paths
VCPKG_SETUP_CMAKE_PROGRAM_PATH:BOOL=ON

//No help, variable specified on the command line.
VCPKG_SET_CHARSET_FLAG:UNINITIALIZED=ON

//No help, variable specified on the command line.
VCPKG_TARGET_ARCHITECTURE:UNINITIALIZED=x64

//Vcpkg target triplet (ex. x86-windows)
VCPKG_TARGET_TRIPLET:STRING=x64-windows

//Trace calls to find_package()
VCPKG_TRACE_FIND_PACKAGE:BOOL=OFF

//Enables messages from the VCPKG toolchain for debugging purposes.
VCPKG_VERBOSE:BOOL=OFF

//(experimental) Automatically copy dependencies into the install
// target directory for executables. Requires CMake 3.14.
X_VCPKG_APPLOCAL_DEPS_INSTALL:BOOL=OFF

//(experimental) Add USES_TERMINAL to VCPKG_APPLOCAL_DEPS to force
// serialization.
X_VCPKG_APPLOCAL_DEPS_SERIALIZED:BOOL=OFF

//Path to a file.
ZLIB_INCLUDE_DIR:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include

//Path to a library.
ZLIB_LIBRARY_DEBUG:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/zlibd.lib

//Path to a library.
ZLIB_LIBRARY_RELEASE:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/zlib.lib

//The directory which contains the installed libraries for each
// triplet
_VCPKG_INSTALLED_DIR:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed

//No help, variable specified on the command line.
_VCPKG_ROOT_DIR:UNINITIALIZED=C:/usr/vcpkg

//Value Computed by CMake
freetype_BINARY_DIR:STATIC=C:/usr/vcpkg/buildtrees/freetype/x64-windows-rel

//Value Computed by CMake
freetype_IS_TOP_LEVEL:STATIC=ON

//Value Computed by CMake
freetype_SOURCE_DIR:STATIC=C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean


########################
# INTERNAL cache entries
########################

//ADVANCED property for variable: BZIP2_INCLUDE_DIR
BZIP2_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: BZIP2_LIBRARY_DEBUG
BZIP2_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: BZIP2_LIBRARY_RELEASE
BZIP2_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//Have symbol BZ2_bzCompressInit
BZIP2_NEED_PREFIX:INTERNAL=1
//ADVANCED property for variable: CMAKE_AR
CMAKE_AR-ADVANCED:INTERNAL=1
//This is the directory where this CMakeCache.txt was created
CMAKE_CACHEFILE_DIR:INTERNAL=c:/usr/vcpkg/buildtrees/freetype/x64-windows-rel
//Major version of cmake used to create the current loaded cache
CMAKE_CACHE_MAJOR_VERSION:INTERNAL=3
//Minor version of cmake used to create the current loaded cache
CMAKE_CACHE_MINOR_VERSION:INTERNAL=25
//Patch version of cmake used to create the current loaded cache
CMAKE_CACHE_PATCH_VERSION:INTERNAL=0
//Path to CMake executable.
CMAKE_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe
//Path to cpack program executable.
CMAKE_CPACK_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cpack.exe
//Path to ctest program executable.
CMAKE_CTEST_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/ctest.exe
//ADVANCED property for variable: CMAKE_C_COMPILER
CMAKE_C_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS
CMAKE_C_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_DEBUG
CMAKE_C_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_MINSIZEREL
CMAKE_C_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELEASE
CMAKE_C_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELWITHDEBINFO
CMAKE_C_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_STANDARD_LIBRARIES
CMAKE_C_STANDARD_LIBRARIES-ADVANCED:INTERNAL=1
//Path to cache edit program executable.
CMAKE_EDIT_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake-gui.exe
//Executable file format
CMAKE_EXECUTABLE_FORMAT:INTERNAL=Unknown
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS
CMAKE_EXE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_DEBUG
CMAKE_EXE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_MINSIZEREL
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELEASE
CMAKE_EXE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXPORT_COMPILE_COMMANDS
CMAKE_EXPORT_COMPILE_COMMANDS-ADVANCED:INTERNAL=1
//Name of external makefile project generator.
CMAKE_EXTRA_GENERATOR:INTERNAL=
//Name of generator.
CMAKE_GENERATOR:INTERNAL=Ninja
//Generator instance identifier.
CMAKE_GENERATOR_INSTANCE:INTERNAL=
//Name of generator platform.
CMAKE_GENERATOR_PLATFORM:INTERNAL=
//Name of generator toolset.
CMAKE_GENERATOR_TOOLSET:INTERNAL=
//Source directory with the top level CMakeLists.txt file for this
// project
CMAKE_HOME_DIRECTORY:INTERNAL=C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean
//ADVANCED property for variable: CMAKE_LINKER
CMAKE_LINKER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS
CMAKE_MODULE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_DEBUG
CMAKE_MODULE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELEASE
CMAKE_MODULE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MT
CMAKE_MT-ADVANCED:INTERNAL=1
//number of local generators
CMAKE_NUMBER_OF_MAKEFILES:INTERNAL=1
//Platform information initialized
CMAKE_PLATFORM_INFO_INITIALIZED:INTERNAL=1
//noop for ranlib
CMAKE_RANLIB:INTERNAL=:
//ADVANCED property for variable: CMAKE_RC_COMPILER
CMAKE_RC_COMPILER-ADVANCED:INTERNAL=1
CMAKE_RC_COMPILER_WORKS:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS
CMAKE_RC_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_DEBUG
CMAKE_RC_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_MINSIZEREL
CMAKE_RC_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_RELEASE
CMAKE_RC_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_RELWITHDEBINFO
CMAKE_RC_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//Path to CMake installation.
CMAKE_ROOT:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/share/cmake-3.25
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS
CMAKE_SHARED_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_DEBUG
CMAKE_SHARED_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELEASE
CMAKE_SHARED_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_INSTALL_RPATH
CMAKE_SKIP_INSTALL_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_RPATH
CMAKE_SKIP_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS
CMAKE_STATIC_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_DEBUG
CMAKE_STATIC_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELEASE
CMAKE_STATIC_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TOOLCHAIN_FILE
CMAKE_TOOLCHAIN_FILE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_VERBOSE_MAKEFILE
CMAKE_VERBOSE_MAKEFILE-ADVANCED:INTERNAL=1
//Details about finding BZip2
FIND_PACKAGE_MESSAGE_DETAILS_BZip2:INTERNAL=[optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/bz2.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/bz2d.lib][C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include][v1.0.8()]
//Details about finding PNG
FIND_PACKAGE_MESSAGE_DETAILS_PNG:INTERNAL=[optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/libpng16.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/libpng16d.lib][C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include][v1.6.38()]
//Details about finding ZLIB
FIND_PACKAGE_MESSAGE_DETAILS_ZLIB:INTERNAL=[optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/zlib.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/zlibd.lib][C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include][v1.2.13()]
PC_BROTLIDEC_CFLAGS:INTERNAL=
PC_BROTLIDEC_CFLAGS_I:INTERNAL=
PC_BROTLIDEC_CFLAGS_OTHER:INTERNAL=
PC_BROTLIDEC_FOUND:INTERNAL=
PC_BROTLIDEC_INCLUDEDIR:INTERNAL=
PC_BROTLIDEC_LIBDIR:INTERNAL=
PC_BROTLIDEC_LIBS:INTERNAL=
PC_BROTLIDEC_LIBS_L:INTERNAL=
PC_BROTLIDEC_LIBS_OTHER:INTERNAL=
PC_BROTLIDEC_LIBS_PATHS:INTERNAL=
PC_BROTLIDEC_MODULE_NAME:INTERNAL=
PC_BROTLIDEC_PREFIX:INTERNAL=
PC_BROTLIDEC_STATIC_CFLAGS:INTERNAL=
PC_BROTLIDEC_STATIC_CFLAGS_I:INTERNAL=
PC_BROTLIDEC_STATIC_CFLAGS_OTHER:INTERNAL=
PC_BROTLIDEC_STATIC_LIBDIR:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS_L:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS_OTHER:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS_PATHS:INTERNAL=
PC_BROTLIDEC_VERSION:INTERNAL=
PC_BROTLIDEC_libbrotlidec_INCLUDEDIR:INTERNAL=
PC_BROTLIDEC_libbrotlidec_LIBDIR:INTERNAL=
PC_BROTLIDEC_libbrotlidec_PREFIX:INTERNAL=
PC_BROTLIDEC_libbrotlidec_VERSION:INTERNAL=
PC_BZIP2_CFLAGS:INTERNAL=
PC_BZIP2_CFLAGS_I:INTERNAL=
PC_BZIP2_CFLAGS_OTHER:INTERNAL=
PC_BZIP2_FOUND:INTERNAL=
PC_BZIP2_INCLUDEDIR:INTERNAL=
PC_BZIP2_LIBDIR:INTERNAL=
PC_BZIP2_LIBS:INTERNAL=
PC_BZIP2_LIBS_L:INTERNAL=
PC_BZIP2_LIBS_OTHER:INTERNAL=
PC_BZIP2_LIBS_PATHS:INTERNAL=
PC_BZIP2_MODULE_NAME:INTERNAL=
PC_BZIP2_PREFIX:INTERNAL=
PC_BZIP2_STATIC_CFLAGS:INTERNAL=
PC_BZIP2_STATIC_CFLAGS_I:INTERNAL=
PC_BZIP2_STATIC_CFLAGS_OTHER:INTERNAL=
PC_BZIP2_STATIC_LIBDIR:INTERNAL=
PC_BZIP2_STATIC_LIBS:INTERNAL=
PC_BZIP2_STATIC_LIBS_L:INTERNAL=
PC_BZIP2_STATIC_LIBS_OTHER:INTERNAL=
PC_BZIP2_STATIC_LIBS_PATHS:INTERNAL=
PC_BZIP2_VERSION:INTERNAL=
//ADVANCED property for variable: PKG_CONFIG_ARGN
PKG_CONFIG_ARGN-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PKG_CONFIG_EXECUTABLE
PKG_CONFIG_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PNG_LIBRARY_DEBUG
PNG_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PNG_LIBRARY_RELEASE
PNG_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PNG_PNG_INCLUDE_DIR
PNG_PNG_INCLUDE_DIR-ADVANCED:INTERNAL=1
//Install the dependencies listed in your manifest:
//\n    If this is off, you will have to manually install your dependencies.
//\n    See https://github.com/microsoft/vcpkg/tree/master/docs/specifications/manifests.md
// for more info.
//\n
VCPKG_MANIFEST_INSTALL:INTERNAL=OFF
//ADVANCED property for variable: VCPKG_VERBOSE
VCPKG_VERBOSE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_INCLUDE_DIR
ZLIB_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_DEBUG
ZLIB_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_RELEASE
ZLIB_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//Making sure VCPKG_MANIFEST_MODE doesn't change
Z_VCPKG_CHECK_MANIFEST_MODE:INTERNAL=OFF
//Vcpkg root directory
Z_VCPKG_ROOT_DIR:INTERNAL=C:/usr/vcpkg
__pkg_config_checked_PC_BROTLIDEC:INTERNAL=1
__pkg_config_checked_PC_BZIP2:INTERNAL=1

```
</details>
<details><summary>C:\usr\vcpkg\buildtrees\freetype\config-x64-windows-CMakeCache.txt.log</summary>

```
# This is the CMakeCache file.
# For build in directory: c:/usr/vcpkg/buildtrees/freetype/x64-windows-rel
# It was generated by CMake: C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe
# You can edit this file to change values found and used by cmake.
# If you do not want to change any of the values, simply exit the editor.
# If you do want to change a value, simply edit, save, and exit the editor.
# The syntax for the file is as follows:
# KEY:TYPE=VALUE
# KEY is the name of a variable in the cache.
# TYPE is a hint to GUIs for the type of VALUE, DO NOT EDIT TYPE!.
# VALUE is the current value for the KEY.

########################
# EXTERNAL cache entries
########################

//Path to a library.
BROTLICOMMON_LIBRARIES:FILEPATH=BROTLICOMMON_LIBRARIES-NOTFOUND

//Path to a file.
BROTLIDEC_INCLUDE_DIRS:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include

//Path to a library.
BROTLIDEC_LIBRARIES:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/brotlidec.lib

//No help, variable specified on the command line.
BUILD_SHARED_LIBS:UNINITIALIZED=ON

//Path to a file.
BZIP2_INCLUDE_DIR:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include

//Path to a library.
BZIP2_LIBRARY_DEBUG:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/bz2d.lib

//Path to a library.
BZIP2_LIBRARY_RELEASE:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/bz2.lib

//Path to a program.
CMAKE_AR:FILEPATH=C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/lib.exe

//Choose the type of build, options are: None Debug Release RelWithDebInfo
// MinSizeRel ...
CMAKE_BUILD_TYPE:STRING=Release

CMAKE_CROSSCOMPILING:STRING=OFF

CMAKE_CXX_FLAGS:STRING=' /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /GR /EHsc /MP '

CMAKE_CXX_FLAGS_DEBUG:STRING='/D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 '

CMAKE_CXX_FLAGS_RELEASE:STRING='/MD /O2 /Oi /Gy /DNDEBUG /Z7 '

//C compiler
CMAKE_C_COMPILER:FILEPATH=C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/cl.exe

CMAKE_C_FLAGS:STRING=' /nologo /DWIN32 /D_WINDOWS /W3 /utf-8 /MP '

CMAKE_C_FLAGS_DEBUG:STRING='/D_DEBUG /MDd /Z7 /Ob0 /Od /RTC1 '

//Flags used by the C compiler during MINSIZEREL builds.
CMAKE_C_FLAGS_MINSIZEREL:STRING=/MD /O1 /Ob1 /DNDEBUG

CMAKE_C_FLAGS_RELEASE:STRING='/MD /O2 /Oi /Gy /DNDEBUG /Z7 '

//Flags used by the C compiler during RELWITHDEBINFO builds.
CMAKE_C_FLAGS_RELWITHDEBINFO:STRING=/MD /Zi /O2 /Ob1 /DNDEBUG

//Libraries linked by default with all C applications.
CMAKE_C_STANDARD_LIBRARIES:STRING=kernel32.lib user32.lib gdi32.lib winspool.lib shell32.lib ole32.lib oleaut32.lib uuid.lib comdlg32.lib advapi32.lib

//No help, variable specified on the command line.
...
Skipped 261 lines
...

// found after toolchain/system libraries/packages.
VCPKG_PREFER_SYSTEM_LIBS:BOOL=OFF

//Enable the setup of CMAKE_PROGRAM_PATH to vcpkg paths
VCPKG_SETUP_CMAKE_PROGRAM_PATH:BOOL=ON

//No help, variable specified on the command line.
VCPKG_SET_CHARSET_FLAG:UNINITIALIZED=ON

//No help, variable specified on the command line.
VCPKG_TARGET_ARCHITECTURE:UNINITIALIZED=x64

//Vcpkg target triplet (ex. x86-windows)
VCPKG_TARGET_TRIPLET:STRING=x64-windows

//Trace calls to find_package()
VCPKG_TRACE_FIND_PACKAGE:BOOL=OFF

//Enables messages from the VCPKG toolchain for debugging purposes.
VCPKG_VERBOSE:BOOL=OFF

//(experimental) Automatically copy dependencies into the install
// target directory for executables. Requires CMake 3.14.
X_VCPKG_APPLOCAL_DEPS_INSTALL:BOOL=OFF

//(experimental) Add USES_TERMINAL to VCPKG_APPLOCAL_DEPS to force
// serialization.
X_VCPKG_APPLOCAL_DEPS_SERIALIZED:BOOL=OFF

//Path to a file.
ZLIB_INCLUDE_DIR:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include

//Path to a library.
ZLIB_LIBRARY_DEBUG:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/zlibd.lib

//Path to a library.
ZLIB_LIBRARY_RELEASE:FILEPATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/zlib.lib

//The directory which contains the installed libraries for each
// triplet
_VCPKG_INSTALLED_DIR:PATH=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed

//No help, variable specified on the command line.
_VCPKG_ROOT_DIR:UNINITIALIZED=C:/usr/vcpkg

//Value Computed by CMake
freetype_BINARY_DIR:STATIC=C:/usr/vcpkg/buildtrees/freetype/x64-windows-rel

//Value Computed by CMake
freetype_IS_TOP_LEVEL:STATIC=ON

//Value Computed by CMake
freetype_SOURCE_DIR:STATIC=C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean


########################
# INTERNAL cache entries
########################

//ADVANCED property for variable: BZIP2_INCLUDE_DIR
BZIP2_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: BZIP2_LIBRARY_DEBUG
BZIP2_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: BZIP2_LIBRARY_RELEASE
BZIP2_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//Have symbol BZ2_bzCompressInit
BZIP2_NEED_PREFIX:INTERNAL=1
//ADVANCED property for variable: CMAKE_AR
CMAKE_AR-ADVANCED:INTERNAL=1
//This is the directory where this CMakeCache.txt was created
CMAKE_CACHEFILE_DIR:INTERNAL=c:/usr/vcpkg/buildtrees/freetype/x64-windows-rel
//Major version of cmake used to create the current loaded cache
CMAKE_CACHE_MAJOR_VERSION:INTERNAL=3
//Minor version of cmake used to create the current loaded cache
CMAKE_CACHE_MINOR_VERSION:INTERNAL=25
//Patch version of cmake used to create the current loaded cache
CMAKE_CACHE_PATCH_VERSION:INTERNAL=0
//Path to CMake executable.
CMAKE_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe
//Path to cpack program executable.
CMAKE_CPACK_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cpack.exe
//Path to ctest program executable.
CMAKE_CTEST_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/ctest.exe
//ADVANCED property for variable: CMAKE_C_COMPILER
CMAKE_C_COMPILER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS
CMAKE_C_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_DEBUG
CMAKE_C_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_MINSIZEREL
CMAKE_C_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELEASE
CMAKE_C_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_FLAGS_RELWITHDEBINFO
CMAKE_C_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_C_STANDARD_LIBRARIES
CMAKE_C_STANDARD_LIBRARIES-ADVANCED:INTERNAL=1
//Path to cache edit program executable.
CMAKE_EDIT_COMMAND:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake-gui.exe
//Executable file format
CMAKE_EXECUTABLE_FORMAT:INTERNAL=Unknown
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS
CMAKE_EXE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_DEBUG
CMAKE_EXE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_MINSIZEREL
CMAKE_EXE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELEASE
CMAKE_EXE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_EXE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_EXPORT_COMPILE_COMMANDS
CMAKE_EXPORT_COMPILE_COMMANDS-ADVANCED:INTERNAL=1
//Name of external makefile project generator.
CMAKE_EXTRA_GENERATOR:INTERNAL=
//Name of generator.
CMAKE_GENERATOR:INTERNAL=Ninja
//Generator instance identifier.
CMAKE_GENERATOR_INSTANCE:INTERNAL=
//Name of generator platform.
CMAKE_GENERATOR_PLATFORM:INTERNAL=
//Name of generator toolset.
CMAKE_GENERATOR_TOOLSET:INTERNAL=
//Source directory with the top level CMakeLists.txt file for this
// project
CMAKE_HOME_DIRECTORY:INTERNAL=C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean
//ADVANCED property for variable: CMAKE_LINKER
CMAKE_LINKER-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS
CMAKE_MODULE_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_DEBUG
CMAKE_MODULE_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL
CMAKE_MODULE_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELEASE
CMAKE_MODULE_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_MODULE_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_MT
CMAKE_MT-ADVANCED:INTERNAL=1
//number of local generators
CMAKE_NUMBER_OF_MAKEFILES:INTERNAL=1
//Platform information initialized
CMAKE_PLATFORM_INFO_INITIALIZED:INTERNAL=1
//noop for ranlib
CMAKE_RANLIB:INTERNAL=:
//ADVANCED property for variable: CMAKE_RC_COMPILER
CMAKE_RC_COMPILER-ADVANCED:INTERNAL=1
CMAKE_RC_COMPILER_WORKS:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS
CMAKE_RC_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_DEBUG
CMAKE_RC_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_MINSIZEREL
CMAKE_RC_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_RELEASE
CMAKE_RC_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_RC_FLAGS_RELWITHDEBINFO
CMAKE_RC_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//Path to CMake installation.
CMAKE_ROOT:INTERNAL=C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/share/cmake-3.25
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS
CMAKE_SHARED_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_DEBUG
CMAKE_SHARED_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL
CMAKE_SHARED_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELEASE
CMAKE_SHARED_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_SHARED_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_INSTALL_RPATH
CMAKE_SKIP_INSTALL_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_SKIP_RPATH
CMAKE_SKIP_RPATH-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS
CMAKE_STATIC_LINKER_FLAGS-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_DEBUG
CMAKE_STATIC_LINKER_FLAGS_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL
CMAKE_STATIC_LINKER_FLAGS_MINSIZEREL-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELEASE
CMAKE_STATIC_LINKER_FLAGS_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO
CMAKE_STATIC_LINKER_FLAGS_RELWITHDEBINFO-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_TOOLCHAIN_FILE
CMAKE_TOOLCHAIN_FILE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: CMAKE_VERBOSE_MAKEFILE
CMAKE_VERBOSE_MAKEFILE-ADVANCED:INTERNAL=1
//Details about finding BZip2
FIND_PACKAGE_MESSAGE_DETAILS_BZip2:INTERNAL=[optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/bz2.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/bz2d.lib][C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include][v1.0.8()]
//Details about finding PNG
FIND_PACKAGE_MESSAGE_DETAILS_PNG:INTERNAL=[optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/libpng16.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/libpng16d.lib][C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include][v1.6.38()]
//Details about finding ZLIB
FIND_PACKAGE_MESSAGE_DETAILS_ZLIB:INTERNAL=[optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/zlib.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/zlibd.lib][C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/include][v1.2.13()]
PC_BROTLIDEC_CFLAGS:INTERNAL=
PC_BROTLIDEC_CFLAGS_I:INTERNAL=
PC_BROTLIDEC_CFLAGS_OTHER:INTERNAL=
PC_BROTLIDEC_FOUND:INTERNAL=
PC_BROTLIDEC_INCLUDEDIR:INTERNAL=
PC_BROTLIDEC_LIBDIR:INTERNAL=
PC_BROTLIDEC_LIBS:INTERNAL=
PC_BROTLIDEC_LIBS_L:INTERNAL=
PC_BROTLIDEC_LIBS_OTHER:INTERNAL=
PC_BROTLIDEC_LIBS_PATHS:INTERNAL=
PC_BROTLIDEC_MODULE_NAME:INTERNAL=
PC_BROTLIDEC_PREFIX:INTERNAL=
PC_BROTLIDEC_STATIC_CFLAGS:INTERNAL=
PC_BROTLIDEC_STATIC_CFLAGS_I:INTERNAL=
PC_BROTLIDEC_STATIC_CFLAGS_OTHER:INTERNAL=
PC_BROTLIDEC_STATIC_LIBDIR:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS_L:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS_OTHER:INTERNAL=
PC_BROTLIDEC_STATIC_LIBS_PATHS:INTERNAL=
PC_BROTLIDEC_VERSION:INTERNAL=
PC_BROTLIDEC_libbrotlidec_INCLUDEDIR:INTERNAL=
PC_BROTLIDEC_libbrotlidec_LIBDIR:INTERNAL=
PC_BROTLIDEC_libbrotlidec_PREFIX:INTERNAL=
PC_BROTLIDEC_libbrotlidec_VERSION:INTERNAL=
PC_BZIP2_CFLAGS:INTERNAL=
PC_BZIP2_CFLAGS_I:INTERNAL=
PC_BZIP2_CFLAGS_OTHER:INTERNAL=
PC_BZIP2_FOUND:INTERNAL=
PC_BZIP2_INCLUDEDIR:INTERNAL=
PC_BZIP2_LIBDIR:INTERNAL=
PC_BZIP2_LIBS:INTERNAL=
PC_BZIP2_LIBS_L:INTERNAL=
PC_BZIP2_LIBS_OTHER:INTERNAL=
PC_BZIP2_LIBS_PATHS:INTERNAL=
PC_BZIP2_MODULE_NAME:INTERNAL=
PC_BZIP2_PREFIX:INTERNAL=
PC_BZIP2_STATIC_CFLAGS:INTERNAL=
PC_BZIP2_STATIC_CFLAGS_I:INTERNAL=
PC_BZIP2_STATIC_CFLAGS_OTHER:INTERNAL=
PC_BZIP2_STATIC_LIBDIR:INTERNAL=
PC_BZIP2_STATIC_LIBS:INTERNAL=
PC_BZIP2_STATIC_LIBS_L:INTERNAL=
PC_BZIP2_STATIC_LIBS_OTHER:INTERNAL=
PC_BZIP2_STATIC_LIBS_PATHS:INTERNAL=
PC_BZIP2_VERSION:INTERNAL=
//ADVANCED property for variable: PKG_CONFIG_ARGN
PKG_CONFIG_ARGN-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PKG_CONFIG_EXECUTABLE
PKG_CONFIG_EXECUTABLE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PNG_LIBRARY_DEBUG
PNG_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PNG_LIBRARY_RELEASE
PNG_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: PNG_PNG_INCLUDE_DIR
PNG_PNG_INCLUDE_DIR-ADVANCED:INTERNAL=1
//Install the dependencies listed in your manifest:
//\n    If this is off, you will have to manually install your dependencies.
//\n    See https://github.com/microsoft/vcpkg/tree/master/docs/specifications/manifests.md
// for more info.
//\n
VCPKG_MANIFEST_INSTALL:INTERNAL=OFF
//ADVANCED property for variable: VCPKG_VERBOSE
VCPKG_VERBOSE-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_INCLUDE_DIR
ZLIB_INCLUDE_DIR-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_DEBUG
ZLIB_LIBRARY_DEBUG-ADVANCED:INTERNAL=1
//ADVANCED property for variable: ZLIB_LIBRARY_RELEASE
ZLIB_LIBRARY_RELEASE-ADVANCED:INTERNAL=1
//Making sure VCPKG_MANIFEST_MODE doesn't change
Z_VCPKG_CHECK_MANIFEST_MODE:INTERNAL=OFF
//Vcpkg root directory
Z_VCPKG_ROOT_DIR:INTERNAL=C:/usr/vcpkg
__pkg_config_checked_PC_BROTLIDEC:INTERNAL=1
__pkg_config_checked_PC_BZIP2:INTERNAL=1

```
</details>
<details><summary>C:\usr\vcpkg\buildtrees\freetype\config-x64-windows-out.log</summary>

```
[1/2] "C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" -E chdir ".." "C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" "C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/usr/vcpkg/packages/freetype_x64-windows" "-DFT_DISABLE_HARFBUZZ=ON" "-DFT_REQUIRE_ZLIB=ON" "-DFT_REQUIRE_BZIP2=ON" "-DFT_ENABLE_ERROR_STRINGS=ON" "-DFT_REQUIRE_PNG=ON" "-DFT_REQUIRE_BROTLI=ON" "-DFT_DISABLE_ZLIB=OFF" "-DFT_DISABLE_BZIP2=OFF" "-DFT_DISABLE_PNG=OFF" "-DFT_DISABLE_BROTLI=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/usr/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON"
FAILED: ../CMakeCache.txt 
"C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" -E chdir ".." "C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" "C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Release" "-DCMAKE_INSTALL_PREFIX=C:/usr/vcpkg/packages/freetype_x64-windows" "-DFT_DISABLE_HARFBUZZ=ON" "-DFT_REQUIRE_ZLIB=ON" "-DFT_REQUIRE_BZIP2=ON" "-DFT_ENABLE_ERROR_STRINGS=ON" "-DFT_REQUIRE_PNG=ON" "-DFT_REQUIRE_BROTLI=ON" "-DFT_DISABLE_ZLIB=OFF" "-DFT_DISABLE_BZIP2=OFF" "-DFT_DISABLE_PNG=OFF" "-DFT_DISABLE_BROTLI=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/usr/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON"
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
-- The C compiler identification is MSVC 19.34.31935.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found ZLIB: optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/zlib.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/zlibd.lib (found version "1.2.13") 
-- Found PNG: optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/libpng16.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/libpng16d.lib (found version "1.6.38") 
-- Found BZip2: optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/bz2.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/bz2d.lib (found version "1.0.8") 
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
CMake Error at C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find BrotliDec (missing: BROTLIDEC_LIBRARIES)
Call Stack (most recent call first):
  C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  builds/cmake/FindBrotliDec.cmake:51 (find_package_handle_standard_args)
  C:/usr/vcpkg/scripts/buildsystems/vcpkg.cmake:838 (_find_package)
  CMakeLists.txt:290 (find_package)


-- Configuring incomplete, errors occurred!
See also "C:/usr/vcpkg/buildtrees/freetype/x64-windows-rel/CMakeFiles/CMakeOutput.log".
[2/2] "C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" -E chdir "../../x64-windows-dbg" "C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" "C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/usr/vcpkg/packages/freetype_x64-windows/debug" "-DFT_DISABLE_HARFBUZZ=ON" "-DFT_REQUIRE_ZLIB=ON" "-DFT_REQUIRE_BZIP2=ON" "-DFT_ENABLE_ERROR_STRINGS=ON" "-DFT_REQUIRE_PNG=ON" "-DFT_REQUIRE_BROTLI=ON" "-DFT_DISABLE_ZLIB=OFF" "-DFT_DISABLE_BZIP2=OFF" "-DFT_DISABLE_PNG=OFF" "-DFT_DISABLE_BROTLI=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/usr/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON"
FAILED: ../../x64-windows-dbg/CMakeCache.txt 
"C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" -E chdir "../../x64-windows-dbg" "C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/bin/cmake.exe" "C:/usr/vcpkg/buildtrees/freetype/src/6482de1748-0955a47a8e.clean" "-G" "Ninja" "-DCMAKE_BUILD_TYPE=Debug" "-DCMAKE_INSTALL_PREFIX=C:/usr/vcpkg/packages/freetype_x64-windows/debug" "-DFT_DISABLE_HARFBUZZ=ON" "-DFT_REQUIRE_ZLIB=ON" "-DFT_REQUIRE_BZIP2=ON" "-DFT_ENABLE_ERROR_STRINGS=ON" "-DFT_REQUIRE_PNG=ON" "-DFT_REQUIRE_BROTLI=ON" "-DFT_DISABLE_ZLIB=OFF" "-DFT_DISABLE_BZIP2=OFF" "-DFT_DISABLE_PNG=OFF" "-DFT_DISABLE_BROTLI=OFF" "-DCMAKE_MAKE_PROGRAM=C:/Program Files/Microsoft Visual Studio/2022/Professional/Common7/IDE/CommonExtensions/Microsoft/CMake/Ninja/ninja.exe" "-DBUILD_SHARED_LIBS=ON" "-DVCPKG_CHAINLOAD_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/toolchains/windows.cmake" "-DVCPKG_TARGET_TRIPLET=x64-windows" "-DVCPKG_SET_CHARSET_FLAG=ON" "-DVCPKG_PLATFORM_TOOLSET=v143" "-DCMAKE_EXPORT_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_PACKAGE_REGISTRY=ON" "-DCMAKE_FIND_PACKAGE_NO_SYSTEM_PACKAGE_REGISTRY=ON" "-DCMAKE_INSTALL_SYSTEM_RUNTIME_LIBS_SKIP=TRUE" "-DCMAKE_VERBOSE_MAKEFILE=ON" "-DVCPKG_APPLOCAL_DEPS=OFF" "-DCMAKE_TOOLCHAIN_FILE=C:/usr/vcpkg/scripts/buildsystems/vcpkg.cmake" "-DCMAKE_ERROR_ON_ABSOLUTE_INSTALL_DESTINATION=ON" "-DVCPKG_CXX_FLAGS=" "-DVCPKG_CXX_FLAGS_RELEASE=" "-DVCPKG_CXX_FLAGS_DEBUG=" "-DVCPKG_C_FLAGS=" "-DVCPKG_C_FLAGS_RELEASE=" "-DVCPKG_C_FLAGS_DEBUG=" "-DVCPKG_CRT_LINKAGE=dynamic" "-DVCPKG_LINKER_FLAGS=" "-DVCPKG_LINKER_FLAGS_RELEASE=" "-DVCPKG_LINKER_FLAGS_DEBUG=" "-DVCPKG_TARGET_ARCHITECTURE=x64" "-DCMAKE_INSTALL_LIBDIR:STRING=lib" "-DCMAKE_INSTALL_BINDIR:STRING=bin" "-D_VCPKG_ROOT_DIR=C:/usr/vcpkg" "-D_VCPKG_INSTALLED_DIR=C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed" "-DVCPKG_MANIFEST_INSTALL=OFF" "-DFETCHCONTENT_FULLY_DISCONNECTED=ON"
-- Could NOT find PkgConfig (missing: PKG_CONFIG_EXECUTABLE) 
-- The C compiler identification is MSVC 19.34.31935.0
-- Detecting C compiler ABI info
-- Detecting C compiler ABI info - done
-- Check for working C compiler: C:/Program Files/Microsoft Visual Studio/2022/Professional/VC/Tools/MSVC/14.34.31933/bin/Hostx64/x64/cl.exe - skipped
-- Detecting C compile features
-- Detecting C compile features - done
-- Found ZLIB: optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/zlib.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/zlibd.lib (found version "1.2.13") 
-- Found PNG: optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/libpng16.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/libpng16d.lib (found version "1.6.38") 
-- Found BZip2: optimized;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/lib/bz2.lib;debug;C:/dev/my-project/build-x86_64-windows-vc143-static/vcpkg_installed/x64-windows/debug/lib/bz2d.lib (found version "1.0.8") 
-- Looking for BZ2_bzCompressInit
-- Looking for BZ2_bzCompressInit - found
CMake Error at C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:230 (message):
  Could NOT find BrotliDec (missing: BROTLIDEC_LIBRARIES)
Call Stack (most recent call first):
  C:/usr/vcpkg/downloads/tools/cmake-3.25.0-windows/cmake-3.25.0-windows-i386/share/cmake-3.25/Modules/FindPackageHandleStandardArgs.cmake:600 (_FPHSA_FAILURE_MESSAGE)
  builds/cmake/FindBrotliDec.cmake:51 (find_package_handle_standard_args)
  C:/usr/vcpkg/scripts/buildsystems/vcpkg.cmake:838 (_find_package)
  CMakeLists.txt:290 (find_package)


-- Configuring incomplete, errors occurred!
See also "C:/usr/vcpkg/buildtrees/freetype/x64-windows-dbg/CMakeFiles/CMakeOutput.log".
ninja: build stopped: subcommand failed.
```
</details>

**Additional context**

<details><summary>vcpkg.json</summary>

```
{
  "name": "my-project",
  "version-string": "1.42.0",
  "dependencies": [
    {
      "name": "ms-gsl"
    },
    {
      "name": "breakpad"
    },
    {
      "name": "catch2"
    },
    {
      "name": "fmt"
    },
    {
      "name": "pcre2"
    },
    {
      "name": "zlib"
    },
    {
      "name": "liblzma"
    },
    {
      "name": "zstd"
    },
    {
      "name": "lz4"
    },
    {
      "name": "lzo"
    },
    {
      "name": "brotli"
    },
    {
      "name": "libarchive",
      "features": [
        "libxml2",
        "bzip2",
        "lz4",
        "lzma",
        "lzo",
        "zstd",
        "openssl"
      ]
    },
    {
      "name": "tesseract"
    },
    {
      "name": "libpng"
    },
    {
      "name": "libjpeg-turbo"
    },
    {
      "name": "openjpeg"
    },
    {
      "name": "libwebp"
    },
    {
      "name": "tiff",
      "features": [
        "cxx",
        "jpeg",
        "lzma",
        "webp",
        "zip",
        "zstd"
      ]
    },
    {
      "name": "libsquish"
    },
    {
      "name": "libsodium"
    },
    {
      "name": "openssl"
    },
    {
      "name": "libtommath"
    },
    {
      "name": "libtomcrypt"
    },
    {
      "name": "cryptopp"
    },
    {
      "name": "libiconv"
    },
    {
      "name": "icu"
    },
    {
      "name": "fribidi"
    },
    {
      "name": "mp3lame"
    },
    {
      "name": "libogg"
    },
    {
      "name": "libvorbis"
    },
    {
      "name": "libtheora"
    },
    {
      "name": "libvpx"
    },
    {
      "name": "ffnvcodec"
    },
    {
      "name": "sdl2",
      "features": [
        "vulkan"
      ]
    },
    {
      "name": "ffmpeg",
      "features": [
        "avcodec",
        "avdevice",
        "avfilter",
        "avformat",
        "bzip2",
        "ffmpeg",
        "ffplay",
        "freetype",
        "fribidi",
        "iconv",
        "lzma",
        "mp3lame",
        "nvcodec",
        "opencl",
        "openjpeg",
        "openssl",
        "swresample",
        "swscale",
        "theora",
        "vorbis",
        "vpx",
        "webp",
        "xml2",
        "zlib"
      ]
    },
    {
      "name": "http-parser"
    },
    {
      "name": "libssh2",
      "features": [
        "zlib"
      ]
    },
    {
      "name": "libgit2",
      "default-features": false,
      "features": [
        "openssl",
        "pcre2"
      ]
    },
    {
      "name": "boost"
    },
    {
      "name": "boost-python",
      "features": [
        "python2"
      ]
    },
    {
      "name": "freetype",
      "default-features": false,
      "features": [
        "brotli",
        "bzip2",
        "error-strings",
        "png",
        "zlib"
      ]
    },
    {
      "name": "lua",
      "features": [
        "cpp",
        "tools"
      ]
    },
    {
      "name": "opencv4",
      "features": [
        "contrib",
        "cuda",
        "cudnn",
        "default-features",
        "freetype",
        "ffmpeg",
        "jpeg",
        "openmp",
        "png",
        "python",
        "quirc",
        "tiff",
        "webp",
        "world"
      ]
    },
    {
      "name": "opencv",
      "features": [
        "contrib",
        "default-features",
        "freetype",
        "ffmpeg",
        "jpeg",
        "openmp",
        "png",
        "world"
      ]
    },
    {
      "name": "qt5",
      "features": [
        "essentials",
        "quickcontrols2"
      ]
    }
  ]
}

```
</details>

--- a/ports/freetype/brotli-static.patch
+++ b/ports/freetype/brotli-static.patch
@@ -2,7 +2,7 @@ diff --git a/builds/cmake/FindBrotliDec.cmake b/builds/cmake/FindBrotliDec.cmake
 index 46356b1fd..ed4cc2409 100644
 --- a/builds/cmake/FindBrotliDec.cmake
 +++ b/builds/cmake/FindBrotliDec.cmake
-@@ -35,10 +35,17 @@ find_path(BROTLIDEC_INCLUDE_DIRS
+@@ -35,10 +35,19 @@ find_path(BROTLIDEC_INCLUDE_DIRS
    PATH_SUFFIXES brotli)
  
  find_library(BROTLIDEC_LIBRARIES
@@ -11,13 +11,15 @@ index 46356b1fd..ed4cc2409 100644
    HINTS ${PC_BROTLIDEC_LIBDIR}
          ${PC_BROTLIDEC_LIBRARY_DIRS})
  
-+if(BROTLIDEC_LIBRARIES MATCHES "-static")
++get_filename_component(z_vcpkg_freetype_brotlidec_name "${BROTLIDEC_LIBRARIES}" NAME)
++if(z_vcpkg_freetype_brotlidec_name MATCHES "-static")
 +  find_library(BROTLICOMMON_LIBRARIES
 +    NAMES brotlicommon-static
 +    HINTS ${PC_BROTLIDEC_LIBDIR}
 +          ${PC_BROTLIDEC_LIBRARY_DIRS})
-+  set(BROTLIDEC_LIBRARIES ${BROTLIDEC_LIBRARIES} ${BROTLICOMMON_LIBRARIES})
++  set(BROTLIDEC_LIBRARIES "${BROTLIDEC_LIBRARIES};${BROTLICOMMON_LIBRARIES}")
 +endif()
++unset(z_vcpkg_freetype_brotlidec_name)
  
  include(FindPackageHandleStandardArgs)
  find_package_handle_standard_args(

--- a/ports/freetype/portfile.cmake
+++ b/ports/freetype/portfile.cmake
@@ -76,7 +76,7 @@ file(REMOVE_RECURSE "${CURRENT_PACKAGES_DIR}/debug/share")
 
 configure_file("${CMAKE_CURRENT_LIST_DIR}/vcpkg-cmake-wrapper.cmake"
     "${CURRENT_PACKAGES_DIR}/share/${PORT}/vcpkg-cmake-wrapper.cmake" @ONLY)
-
+file(INSTALL "${CMAKE_CURRENT_LIST_DIR}/usage" DESTINATION "${CURRENT_PACKAGES_DIR}/share/${PORT}")
 vcpkg_install_copyright(
     FILE_LIST
         "${SOURCE_PATH}/LICENSE.TXT"

--- a/ports/freetype/usage
+++ b/ports/freetype/usage
@@ -1,0 +1,4 @@
+freetype is compatible with built-in CMake targets:
+
+    find_package(Freetype REQUIRED)
+    target_link_libraries(main PRIVATE Freetype::Freetype) # since CMake 3.10

--- a/ports/freetype/vcpkg.json
+++ b/ports/freetype/vcpkg.json
@@ -1,7 +1,7 @@
 {
   "name": "freetype",
   "version": "2.12.1",
-  "port-version": 2,
+  "port-version": 3,
   "description": "A library to render fonts.",
   "homepage": "https://www.freetype.org/",
   "license": "FTL OR GPL-2.0-or-later",

--- a/versions/baseline.json
+++ b/versions/baseline.json
@@ -2454,7 +2454,7 @@
     },
     "freetype": {
       "baseline": "2.12.1",
-      "port-version": 2
+      "port-version": 3
     },
     "freetype-gl": {
       "baseline": "2022-01-17",

--- a/versions/f-/freetype.json
+++ b/versions/f-/freetype.json
@@ -1,6 +1,11 @@
 {
   "versions": [
     {
+      "git-tree": "db92ea87ff12339808263710675221df6a06ac15",
+      "version": "2.12.1",
+      "port-version": 3
+    },
+    {
       "git-tree": "f1d6772ea2cb8cd74146b263b5dab534a10dd641",
       "version": "2.12.1",
       "port-version": 2

